doc/src/user_guide.rst

```diff
@@ -1570,6 +1570,80 @@ Example::
     int-E = rho*(u*u + v*v + w*w)
     int-enst = rho*(%(vor1)s*%(vor1)s + %(vor2)s*%(vor2)s + %(vor3)s*%(vor3)s)
 
+.. _ascent-plugin:
+
+[soln-plugin-ascent]
+^^^^^^^^^^^^^^^^^^^^
+
+Uses `Alpine Ascent <https://github.com/Alpine-DAV/ascent>`_ to plot on-the-fly
+. You will need both Conduit and Ascent compiled with MPI and VTK-h support. The
+paths to the libraries will need to be set as environment variables, for
+example on linux via::
+
+    export PYFR_CONDUIT_LIBRARY_PATH=/path/to/libconduit.so
+    export PYFR_ASCENT_MPI_LIBRARY_PATH=/path/to/libascent_mpi.so
+
+The following parameters can then be set:
+
+1. ``nsteps`` --- produce the plots every ``nsteps`` time steps:
+
+    *int*
+
+2. ``division`` --- the level of linear subdivison to use
+
+    *int*
+
+3. ``region`` --- region to integrate, specified as either the
+   entire domain using ``*`` or a combination of the geometric shapes
+   specified in :ref:`regions`:
+
+    ``*`` | ``shape(args, ...)``
+
+There are then three components that can be used to build plots. Scenes which
+actualy define the render, Pipelines that can be used to apply filers and build
+sequenceces of data manipulations, and Fields which be used to define field
+expresions.
+
+1. ``field-{name}`` --- this is an extension to the Ascent library where users
+define expressions for the fields used. This can either be a scalar or a
```

Comment:
Should we make explicit what you have to play with here? Is it only the primitive variables rho, u, v, w, p? Can you use x, y, z, and t as well?